kso_utils/MLProjectProcessor.py

```diff
@@ -49,6 +47,20 @@ def __init__(
         self.model_type = 1  # set as 1 for testing
         self.train, self.run, self.test = (None,) * 3
         self.registry = "wandb"  # TODO: make this a config and read it in here instead of hardcoding
+        self.registry_utils = import_module(f"kso_utils.{self.registry}_utils")
```

Comment:
Let's try to improve the structure here a bit:
- Create a "registries" folder under kso_utils
- Place a wandb.py file in there, without the "utils"

Then it's a lot more obvious what further registries have to implement. And let's not call this "registry_utils", just `self.registry`. The `_utils` suffix provides no real value, if you need to rename something, then rename the string value, now that it's not used anywhere else